external-import/intel471_v2/entrypoint.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+# Correct working directory
+cd /opt/opencti-intel471
```

Comment:
@mmolenda Could you change it and add v2 please to not having problem with v1 of connector ?